.github/scripts/workflow-monitor.py

```diff
@@ -17,10 +17,10 @@
 import sys
 import requests
 
-from common import terminate_workflow_instances, stop_workflow_instances, gha_runs_api_url
-
+from common import get_platform_lib, gha_runs_api_url
+from platform_lib import Platform, get_platform_enum
 # Time between HTTPS requests to github
-POLLING_INTERVAL_SECONDS = 60
+POLLING_INTERVAL_SECONDS = 5
```

Comment:
Just checking, does this have any noticeable impact on CI performance? It shouldn't but we were very conservative in the past.